src/Util/ProjectUpdateTrait.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace UpdateTool\Util;
+
+use UpdateTool\Git\Remote;
+use UpdateTool\Git\WorkingCopy;
+use UpdateTool\Util\SupportLevel;
+
+trait ProjectUpdateTrait
+{
+
+    /**
+     * Update project info (codeowners and support level badge).
+     */
+    protected function updateProjectInfo($api, $project, $baseBranch, $branchName, $commitMessage, $prTitle, $prBody, $logger, $supportLevelBadge = '', $codeowners = '')
```

Comment:
This seems a little heavyweight for a trait, especially with the logger injection. Instead, make this a utility class, and inject the logger via the constructor.

Reply:
Or use setLogger like workingCopy does.